src/components/formComponentTemplateEndpoints.tsx

```diff
@@ -678,6 +679,35 @@ export const formComponentTemplateEndpoints = (
         ),
         validate: validate.xdr,
       };
+    case "filters":
+      return {
+        render: (templ: TemplateRenderTxProps) => (
+          <FiltersPicker
+            key={id}
+            id={id}
+            value={parseJsonString(templ.value)}
+            error={templ.error}
+            onChange={templ?.onChange}
+          />
+        ),
+        validate: null,
```

Comment:
Just checking if we don't need validation here. Is it OK to submit empty fields here?

![image](https://github.com/stellar/laboratory/assets/7346473/5116f31a-3241-498d-af3f-a8374adca068)